ansible-ipi-install/roles/post-install/tasks/main.yml

```diff
@@ -0,0 +1,114 @@
+- name: Wait 7m for all the nodes to settle
+  command: sleep 900
+
+- block:
+  - name: Label infra nodes
+    shell: |
+       oc label node '{{ infra_node_string_1 }}' 'node-role.kubernetes.io/infra=' --overwrite 
+
+  - name: Label infra nodes
+    shell: |
+       oc label node '{{ infra_node_string_2 }}' 'node-role.kubernetes.io/infra=' --overwrite
+
+  - name: Label workload nodes
+    shell: |
+       oc label node '{{ workload_node_string }}' 'node-role.kubernetes.io/workload=' --overwrite
+  when: infra_node_string_1 != "" or infra_node_string_2 != "" or workload_node_string != ""
+
+- name: OVN block
+  block:
+  - name: Scale CVO Down for OVNKubernetes image patching
+    command: oc scale -n openshift-cluster-version deployment.apps/cluster-version-operator --replicas=0
+  
+  - name: Patch new OVNKubernetes Image
+    command: oc -n openshift-network-operator set env deployment.apps/network-operator OVN_IMAGE={{openshift_ovn_image}} RELEASE_VERSION="5.0.0"
+  
+  - name: Pause the rollout to begin
+    pause:
+      seconds: 10
+  
+  - name: Wait for OVNK new node images to roll out
+    command: oc rollout status -n {{ ovn_namespace }} ds/ovnkube-node
+  
+  - name: Wait for OVNK new master images to roll out
+    command: oc rollout status -n {{ ovn_namespace }} ds/ovnkube-master
+  
+  - name: OVN SBDB block
+    block:
+    - name: Deploy ovnkube-sbdb-relay
+      script: sbdb-relay.sh deploy
+    
+    - name: Wait for OVNK sbdb-relay images to roll out
+      command: oc rollout status -n {{ ovn_namespace }} deployment.apps/ovnkube-sbdb-relay
+    
+    - name: Verify ovnkube-sbdb-relay connectivity
+      script: sbdb-relay.sh verify
+    when: openshift_toggle_ovn_relay|bool
```

Comment:
I think we should remove this block. In addition, the openshift_toggle_ovn_relay variable is not declared and the sbdb-relay.sh script is not here

Reply:
right, updated